docs/bigquery/snippets.py

```diff
@@ -2153,22 +2153,28 @@ def test_undelete_table(client, to_delete):
 
     table_ref = client.dataset(dataset_id).table(table_id)
 
-    # Record the current time in milliseconds. We'll use this as the snapshot
-    # time for recovering the table.
-    snapshot_time = int(time.time() * 1000)
+    # TODO(Developer):  Choose an appropriate snapshot point as epoch
+    # milliseconds.
+    #
+    # For this example, we choose the current time as we're about to delete
+    # the table immediately afterwards.
+    snapshot_epoch = int(time.time() * 1000)
+    # [END bigquery_undelete_table]
 
-    # Because local system time may have some drift, we ensure the snapshot
-    # time is at least within the creation lifespan of the table.  Trying
-    # to snapshot a table prior to its creation time is an error.
+    # Due to very short lifecycle of the table, ensure we're not picking a time
+    # prior to the table creation due to time drift between backend and client.
     table = client.get_table(table_ref)
-    if table.created_time > snapshot_time:
-      snapshot_time = table.created_time
+    created_epoch = table._properties['creationTime']
```

Comment:
`table.created` converts the `creationTime` property to a `datetime.datetime` (in UTC).